src/components/LocationSelector/LocationSelector.tsx

```diff
@@ -77,18 +144,19 @@ export function LocationSelector(props) {
      * if this location was not selected yet
      */
     function handleAddLocation(location) {
-        const { Geocoder } = window.google.maps;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const { Geocoder } = (window as any).google.maps;
         const geocoder = new Geocoder();
 
         return findCenter(geocoder, location.place_id)
-            .then((center) => {
+            .then((center: any) => {
```

Comment:
Can you use `CenterClass` instead of `any`?
https://github.com/textkernel/oneui/pull/255/files#diff-299f591a5c6c88980bc5d24d2e8736baR26

Reply:
Apparently `CenterClass` is a value and cannot be used as a type here. Maybe we should check for an additional package which would add google maps types to the project but I think it would be better to do that in the next story.

![image](https://user-images.githubusercontent.com/24496348/82299199-82c06a80-99b5-11ea-81b4-9cc790e6cf42.png)